python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -174,14 +174,15 @@ def run(self, sources, exposure, posExp=None, negExp=None, **kwargs):
         **kwargs
             Additional keyword arguments for `lsst.meas.base.sfm.SingleFrameMeasurementTask`.
         """
+        self.plugins = self.plugins_pre
```

Comment:
You describe what you're doing in the description above, but please add a code comment here and before the next `self.plugins...` line below stating what each `super().run(...` call is doing. I can see future developers getting confused.